Gemfile

```diff
@@ -50,12 +61,16 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'debug', platforms: %i[mri mingw x64_mingw]
+end
+
+group :test do
+  gem 'rspec-rails'
 end
```

Comment:
- [ ] Kindly move `gem 'rspec-rails'` to `group :development, :test`  and delete this `group :test` because it already exist on line `82` to avoid repeatition.